rolling-update with empty file panics

For example
```
> touch empty
> kubectl rolling-update rcname  -f empty

panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x19dca80, 0xc82000e0d0)
        /usr/local/go/src/runtime/panic.go:481 +0x3e6
k8s.io/kubernetes/pkg/kubectl/cmd.RunRollingUpdate(0xc8204c6200, 0x7fda65fb6198, 0xc820090008, 0xc820324800, 0xc8200a4690, 0x1, 0x5, 0xc82029f380, 0x0, 0x0)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/rollingupdate.go:220 +0x46ee
k8s.io/kubernetes/pkg/kubectl/cmd.NewCmdRollingUpdate.func1(0xc820324800, 0xc8200a4690, 0x1, 0x5)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/rollingupdate.go:86 +0x7d
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).execute(0xc820324800, 0xc8200a4640, 0x5, 0x5, 0x0, 0x0)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:565 +0x85a
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc8200d2000, 0xc820324800, 0x0, 0x0)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:651 +0x55c
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).Execute(0xc8200d2000, 0x0, 0x0)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:610 +0x2d
k8s.io/kubernetes/cmd/kubectl/app.Run(0x0, 0x0)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubectl/app/kubectl.go:32 +0xf0
main.main()
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubectl/kubectl.go:28 +0x28
```

--- a/pkg/kubectl/cmd/rollingupdate.go
+++ b/pkg/kubectl/cmd/rollingupdate.go
@@ -220,6 +220,9 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 			if len(list.Items) > 1 {
 				return cmdutil.UsageError(cmd, "%s specifies multiple items", filename)
 			}
+			if len(list.Items) == 0 {
+				return cmdutil.UsageError(cmd, "please make sure %s exists and is not empty", filename)
+			}
 			obj = list.Items[0]
 		}
 		newRc, ok = obj.(*api.ReplicationController)

